newswires/client/src/SearchSummary.tsx

```diff
@@ -291,12 +291,15 @@ const Summary = ({
 	);
 };
 
-export const SearchSummary = () => {
+export const SearchSummary = ({
```

Comment:
The use of `setSideNavIsOpen` in the search summary here (for ticker layouts) is the reason why we need to raise the side nav state up. It would be nice to restructure the layout so that we don't need this, but that would be a bigger change.